msp/identities.go

```diff
@@ -107,6 +108,17 @@ func (id *identity) Validate() error {
 	return id.msp.Validate(id)
 }
 
+type OUIDs []*OUIdentifier
+
+func (o OUIDs) String() string {
+	var res []string
+	for _, id := range o {
+		res = append(res, id.OrganizationalUnitIdentifier)
```

Comment:
You might also want to print a hex encoding of CertifiersIdentifier

Reply:
thx for review! i wonder how that would help debugging log message?

Reply:
The presence of CertifiersIdentifier determines which certification paths can issue valid OUs, so it will be helpful to have that information imho

Reply:
updated it to include a short hex code

Reply:
Thanks, LGTM